features/step_definitions/user.rb

```diff
@@ -72,3 +72,51 @@
 Then(/^(.*) should not be able to sign up$/) do |username|
   expect(user_named(username).persisted?).to be_falsey
 end
+
+
+When(/^(.+) visits the password recovery page$/) do |username|
```

Comment:
since this action does not require any named object as reference, may you should get rid of the parameter, maybe 'When he visits...'.

Reply:
:+1: